test/Microsoft.AspNetCore.Mvc.Formatters.Xml.Test/Internal/MvcXmlDataContractSerializerMvcOptionsSetupTest.cs

```diff
@@ -1,42 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using Microsoft.Extensions.Logging.Abstractions;
-using Xunit;
-
-namespace Microsoft.AspNetCore.Mvc.Formatters.Xml.Internal
```

Comment:
This type was renamed to https://github.com/aspnet/Mvc/pull/8529/files#diff-e7a152988018dcb2d8180cb94a414e95R8